docs/UserGuide.md

```diff
@@ -31,7 +31,7 @@ It is a convenient platform for you to keep track of your lessons and assignment
 
    * **`list`** : Lists all assignments.
 
-   * **`add`**`n/Lab report 3 d/23/04/2020 12:30 PM mod/CS2100` : Adds an assignment named `Lab Assignment 3` to the Address Book.
+   * **`add`**`n/Lab report 3 d/23/04/2020 t/1230 mod/CS2100` : Adds an assignment named `Lab report 3` to your schedule.
```

Comment:
Maybe including the time the assignment is due in the deadline would be better?
E.g: n/Lab report 3 **d/23/04/2020 1230** mod/CS2100

Reply:
okay we can discuss this in our meeting later!